examples/ecs/ambiguity_detection.rs

```diff
@@ -1,26 +1,28 @@
-//! A test to confirm that `bevy` doesn't have system order ambiguity with DefaultPlugins
+//! A test to confirm that `bevy` doesn't have system order ambiguity with [`DefaultPlugins`]
```

Comment:
it's not a test anymore. Also, the example should be listed accordingly in readme ; maybe there's a script for that?